dizoo/d4rl/config/hopper_medium_expert_ibc_ar_config.py

```diff
@@ -0,0 +1,62 @@
+from easydict import EasyDict
+
+cuda = True
+
+main_config = dict(
+    exp_name='hopper_medium_expert_ibc_ar_seed0',
+    env=dict(
+        env_id='hopper-medium-expert-v0',
+        evaluator_env_num=8,
+        use_act_scale=True,
+        n_evaluator_episode=8,
+        stop_value=6000,
+    ),
+    policy=dict(
+        cuda=cuda,
+        model=dict(
+            type='arebm',
+            import_names=['ding.model.template.ebm'],
+            obs_shape=11,
+            action_shape=3,
+            cuda=cuda,
+            stochastic_optim=dict(type='ardfo', cuda=cuda,)
+        ),
+        learn=dict(
+            holdout_ratio=0.1,
+            train_epoch=300,
+            batch_size=256,
+            optim=dict(
+                learning_rate=1e-5,
+                weight_decay=0.0,
+                beta1=0.9,
```

Comment:
why not use default value in policy default_config

Reply:
changed